src/test/java/teammates/test/cases/webapi/DeleteFeedbackResponseActionTest.java

```diff
@@ -20,16 +42,244 @@ protected String getRequestMethod() {
+        };
+        verifyCannotAccess(moderatedStudentSubmissionParams);
+
+        ______TS("non-existent feedback response");
+
+        loginAsStudent(student1InCourse1.getGoogleId());
+        String[] nonExistParams = new String[] {
+                Const.ParamsNames.INTENT, Intent.STUDENT_SUBMISSION.toString(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID, "randomNonExistId",
+        };
+
+        assertThrows(EntityNotFoundException.class, () -> getAction(nonExistParams).checkAccessControl());
+
+        ______TS("Student intends to access other person's response, should not be accessible");
+
+        loginAsStudent(student2InCourse1.getGoogleId());
+        String[] studentAccessOtherPersonParams = new String[] {
+                Const.ParamsNames.INTENT, Intent.STUDENT_SUBMISSION.toString(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID, typicalResponse.getId(),
+        };
+
+        verifyCannotAccess(studentAccessOtherPersonParams);
+
+        ______TS("Student intends to access own response, should be accessible");
+
+        loginAsStudent(student1InCourse1.getGoogleId());
+        String[] studentAccessOwnPersonParams = new String[] {
+                Const.ParamsNames.INTENT, Intent.STUDENT_SUBMISSION.toString(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID, typicalResponse.getId(),
+        };
+
+        verifyCanAccess(studentAccessOwnPersonParams);
+
+        ______TS("Student intends to access same team's response, should be accessible");
+
+        loginAsStudent(student1InCourse1.getGoogleId());
+
+        String[] studentAccessOSameTeamParams = new String[] {
+                Const.ParamsNames.INTENT, Intent.STUDENT_SUBMISSION.toString(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID, typicalResponse2.getId(),
+        };
+
+        verifyCanAccess(studentAccessOSameTeamParams);
+
+        ______TS("Student intends to access other team's response, should not be accessible");
+
+        loginAsStudent(student5InCourse1.getGoogleId());
+
+        String[] studentAccessOtherTeamParams = new String[] {
+                Const.ParamsNames.INTENT, Intent.STUDENT_SUBMISSION.toString(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID, typicalResponse2.getId(),
+        };
+
+        verifyCannotAccess(studentAccessOtherTeamParams);
+
+        ______TS("Instructor intends to access other person's response, should not be accessible");
+
+        loginAsInstructor(instructor2OfCourse1.getGoogleId());
+
+        String[] instructorAccessOtherPersonParams = new String[] {
+                Const.ParamsNames.INTENT, Intent.INSTRUCTOR_SUBMISSION.toString(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID, typicalResponse3.getId(),
+        };
+
+        verifyCannotAccess(instructorAccessOtherPersonParams);
+
+        ______TS("Instructor intends to access own response, should not be accessible");
```

Comment:
`.. should not be accessible ..`, but the code is `verifyCanAccess(instructorAccessOwnPersonParams);`?